useful-scripts/importer-tests/cleanup-test-files.py

```diff
@@ -0,0 +1,13 @@
+import os 
+from pathlib import Path
+
+# simple function to cleanup test zip files 
+def clean_zip_files():
+    for ZIP_FILE in Path('.').glob('test-*.zip'):
```

Comment:
Python convention is for variable names to be lowercase, so ZIP_FILE should be zip_file here.

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names